101-datastore-create-sql/azuredeploy.json

```diff
@@ -0,0 +1,121 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"workspaceName": {
+			"type": "string",
+			"metadata": {
+				"description": "Specifies the name of the Azure Machine Learning workspace which will hold this datastore target."
+			}
+		},
+		"datastoreName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the datastore, case insensitive, can only contain alphanumeric characters and underscore"
+			}
+		},
+		"sqlServerName": {
+			"type": "string",
+			"metadata": {
+				"description": "The SQL server name."
+			}
+		},
+		"authenticationType": {
+			"type": "string",
+			"defaultValue": "SQL Authentication",
+			"allowedValues": [
+				"SQL Authentication",
+				"Service principal"
+			],
+			"metadata": {
+				"description": "Authentication type"
+			}
+		},
+		"userNameOrClientId": {
+			"type": "string",
+			"metadata": {
+				"description": "The username of the database user or service principal/application ID."
+			}
+		},
+		"passwordOrClientSecret": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The password of the database user or service principal's secret."
+			}
+		},
+		"tenantId": {
+			"type": "string",
+			"defaultValue": "[if(equals(parameters('authenticationType'), 'Service principal'), subscription().tenantId, '')]",
+			"metadata": {
+				"description": "This is ignored if SQL Authentication is selected."
+			}
+		},
+		"databaseName": {
+			"type": "string",
+			"metadata": {
+				"description": "The database name."
+			}
+		},
+		"authorityUrl": {
+			"type": "string",
+			"defaultValue": "https://login.microsoftonline.com",
```

Comment:
Are there other authn methods we support?

If we do need this, use the environment() function to get the value (you cannot hardcode it)

Reply:
We can remove these